packages/stream_chat/lib/src/client.dart

```diff
@@ -1350,9 +1352,11 @@ class StreamChatClient {
     if (timeoutOrExpirationDate is DateTime) {
       pinExpires = timeoutOrExpirationDate.toUtc();
     } else if (timeoutOrExpirationDate is num) {
-      pinExpires = DateTime.now().add(
-        Duration(seconds: timeoutOrExpirationDate.toInt()),
-      );
+      pinExpires = DateTime.now()
+          .add(
+            Duration(seconds: timeoutOrExpirationDate.toInt()),
+          )
+          .toUtc();
```

Comment:
This wasn't necessary btw, as we already convert to UTC in Message constructor.

Reply:
In that case the top one would be redundant too, right?